src/components/MovieDetails/index.tsx

```diff
@@ -398,89 +398,91 @@ const MovieDetails: React.FC<MovieDetailsProps> = ({ movie }) => {
           {(trailerUrl ||
             data.mediaInfo?.plexUrl ||
             data.mediaInfo?.plexUrl4k) && (
-            <ButtonWithDropdown
-              buttonType="ghost"
-              text={
-                <>
-                  <svg
-                    className="w-5 h-5 mr-1"
-                    fill="none"
-                    stroke="currentColor"
-                    viewBox="0 0 24 24"
-                    xmlns="http://www.w3.org/2000/svg"
-                  >
-                    <path
-                      strokeLinecap="round"
-                      strokeLinejoin="round"
-                      strokeWidth={2}
-                      d="M14.752 11.168l-3.197-2.132A1 1 0 0010 9.87v4.263a1 1 0 001.555.832l3.197-2.132a1 1 0 000-1.664z"
-                    />
-                    <path
-                      strokeLinecap="round"
-                      strokeLinejoin="round"
-                      strokeWidth={2}
-                      d="M21 12a9 9 0 11-18 0 9 9 0 0118 0z"
-                    />
-                  </svg>
-                  <span>
-                    {data.mediaInfo?.plexUrl
-                      ? intl.formatMessage(messages.playonplex)
-                      : data.mediaInfo?.plexUrl4k &&
-                        (hasPermission(Permission.REQUEST_4K) ||
-                          hasPermission(Permission.REQUEST_4K_MOVIE))
-                      ? intl.formatMessage(messages.playonplex)
-                      : intl.formatMessage(messages.watchtrailer)}
-                  </span>
-                </>
-              }
-              onClick={() => {
-                if (data.mediaInfo?.plexUrl) {
-                  window.open(data.mediaInfo?.plexUrl, '_blank');
-                } else if (data.mediaInfo?.plexUrl4k) {
-                  window.open(data.mediaInfo?.plexUrl4k, '_blank');
-                } else if (trailerUrl) {
-                  window.open(trailerUrl, '_blank');
+            <div className="mb-3 md:mb-0">
```

Comment:
Instead of wrapping `ButtonWithDropdown` in a `div`, you can simply give it a `className` attribute.  Also, it should be `mb-3 sm:mb-0` instead of `mb-3 md:mb-0`.